static/app/stores/groupingStore.tsx

```diff
@@ -47,14 +47,44 @@ type State = {
 
 type ScoreMap = Record<string, number | null>;
 
+export type ApiFingerprint = {
```

Comment:
since this type is only being used in this file, we can remove the `export`

```suggestion
type ApiFingerprint = {
```